modules/arcgis/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@deck.gl/arcgis",
+  "version": "0.0.1",
+  "main": "index.js",
+  "devDependencies": {
+    "@theintern/istanbul-loader": "^1.0.0-beta.2",
+    "@types/sinon": "^7.0.13",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "~5.0.3",
+    "css-loader": "^3.0.0",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.7.0",
+    "node-sass": "~4.12.0",
+    "prettier": "^1.14.3",
+    "resolve-url-loader": "^2.3.1",
+    "sass-loader": "^7.1.0",
+    "style-loader": "~0.23.1",
+    "terser-webpack-plugin": "^1.3.0",
+    "webpack": "^4.20.1",
+    "webpack-cli": "~3.3.5",
+    "webpack-dev-server": "^3.1.10"
+  },
+  "license": "Apache-2.0",
+  "scripts": {},
+  "dependencies": {
+    "@arcgis/webpack-plugin": "~4.13.0",
```

Comment:
Out of curiosity, what does this plugin do, why is it needed as a dependency in an already built library?

Reply:
The ArcGIS API is based on Dojo. For the time being we need the plugin to load the API in Webpack. I think the plugin creates a lightweight AMD loader that sits behind the usual Webpack module loading system.